src/GafferSceneUI/SceneView.cpp

```diff
@@ -1045,21 +1045,22 @@ struct FreeCamera
 	{
 		CameraPtr camera = new Camera;
 		camera->setProjection( "orthographic" );
-		camera->setClippingPlanes( V2f( -100000, 100000 ) );
+		camera->setClippingPlanes( V2f( 0, 100000 ) );
 		camera->setAperture( V2f( 20 ) );
 		camera->setFilmFit( Camera::FilmFit::Fit );
 
+		const float offset = 1000;
```

Comment:
> How was this offset chosen?

Pretty arbitrarily!

I _did_ have a reason for choosing the specific aperture I've gone for, which is that it gives sensible framing of the default grid, so feels in the right scale with the default perspective camera. But the offset was more of a guess.

> If the goal is to not be outside the wall of a room, 1000 units seems like it would usually be too high?

That wasn't the goal - I was aiming to always be outside an asset of reasonable size, but without being so far back that the far clip might come into play for a larger asset. If there's a more principled way of arriving at a number for those goals, I'd be  happy to use it.

In the case of a building with an interior, I don't think there's any way of knowing if an interior is more suitable for the default framing from an exterior, and choosing the exterior has a higher chance of orienting the user well. The near clop of 0 and the framing operation (<kbd>F</kbd>) is designed to be effective in taking you into an interior when you want.

Reply:
Oh, I just got lost somehow while reading this - I was somehow thinking this offset was applied while framing, which would be weird.  Having it for the default position is fine.  Oops.